tests/lax_control_flow_test.py

```diff
@@ -416,6 +416,34 @@ def false_fun(x):
     self.assertEqual(fun(4), cfun(4))
     self.assertEqual(fun(4), (8, 16))
 
+  def testIssue1379(self):
+    # integer predicates should be treated as boolean by testing whether they are zero
+    def fun(x):
```

Comment:
I'm not quite sure what the point of this function is -- it looks like it's testing Python itself?

Reply:
This is used to test that Python and jax create same outputs.

Reply:
I have updated tests. Now the tests are testing regular lax vs jit compiled lax function calls.